### ISSUE TITLE
Dash constructor args and config document section

### ISSUE THREAD

**Issue by byronz** [plotly/dash-docs#505]
this is a placeholder to follow up https://github.com/plotly/dash/issues/746. right now we don't have a full reference of dash.Dash arguments and/or configs to users. we should have a dedicated section in dash docs after the work in 746.

**Comment by maulberto3** [comment]
I believe this is pending right?

**Comment by alexcjohnson** [comment]
Correct, still pending. If you're interested in helping out, I'm thinking maybe we should add a page called something like "Dash class reference" in the "Beyond the Basics" section (is there a better place than that?). This page would include not only the constructor and config, but also the various methods and attributes that might be useful.
